src/main/java/frc/robot/constants/ElevatorConstants.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.constants;
+
+import edu.wpi.first.math.util.Units;
+
+public class ElevatorConstants {
+  //TODO: Set everything to meters in whole robot code
```

Comment:
if it is for the whole robot don't put it here

Reply:
sorry, this was already done

Reply:
I forgot to delete the comment